docs/guides/upgrade-guide.md

```diff
@@ -41,6 +41,33 @@ The new icons automatically sync with theme changes, support all InstUI color to
 
 ## API Changes
 
+### Focus rings
+
+Focus rings are now styled in a central pseudo-component called `SharedTokens`. You can override here how it looks, for example:
+
+```js
+---
+type: example
+---
+<InstUISettingsProvider
+  theme={{
+  themeOverrides: {
+    canvas: {
+      componentOverrides: {
+        SharedTokens: {
+          focusOutline: {
+            infoColor: 'red',
+            offset: '2rem'
+          }
+        }
+      }
+    }
+  }
+}}>
+  <TextInput renderLabel="Name" placeholder="Doe, John Doe"/>
+</InstUISettingsProvider>
+```
```

Comment:
Something is wrong with the overrides, this example is not working. I'll create a ticket to fix it